addons/event_sale/migrations/11.0.1.1/pre-migration.py

```diff
@@ -0,0 +1,9 @@
+# Copyright 2018 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    env.ref('event_sale.event_type').unlink()
```

Comment:
You should enclose it in a try and with savepoint, because it can be in use:
```python
try:
    with env.cr.savepoint():
        env.ref('event_sale.event_type').unlink()
except Exception:
    pass
```